invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/theme.js

```diff
@@ -115,3 +115,19 @@ if (headerSearchbar) {
     headerSearchbar
   );
 }
+
+// Login Logout Button
+const $authButton = $(".auth-button");
```

Comment:
@sakshamarora1 do you have an ID for these ones, to improve performance?
E.g. `$("#login-box").find(".auth-button")`.

Searching in the entire dom for a class could be very inefficient.